.travis.yml

```diff
@@ -1,8 +1,11 @@
 notifications:
   email: false
 
+before_install:
+  - gem update --system 2.6.9 #https://github.com/sickill/rainbow/issues/44
```

Comment:
Can we just use latest? (i.e. https://github.com/projectblacklight/blacklight/blob/master/.travis.yml#L21)